Content.Client/UserInterface/Systems/Character/CharacterUIController.cs

```diff
@@ -221,18 +221,11 @@ private void UpdateRoleType()
         if (!_ent.TryGetComponent<MindComponent>(container.Mind.Value, out var mind))
             return;
 
-        var roleText = Loc.GetString("role-type-crew-aligned-name");
-        var color = Color.White;
-        if (_prototypeManager.TryIndex(mind.RoleType, out var proto))
-        {
-            roleText = Loc.GetString(proto.Name);
-            color = proto.Color;
-        }
-        else
-            _sawmill.Error($"{_player.LocalEntity} has invalid Role Type '{mind.RoleType}'. Displaying '{roleText}' instead");
+        if (!_prototypeManager.TryIndex(mind.RoleType, out var proto))
+            _sawmill.Error($"{_player.LocalEntity} has invalid Role Type '{mind.RoleType}'. Displaying default instead");
```

Comment:
This error will just print the client entity ID to the console. This isn't really that bad, as the RoleType will still be what matters, but we can't really do anything with the client entity ID.

Reply:
Replaced it with player local session string